server/controllers/auth.js

```diff
@@ -92,6 +93,8 @@ exports.loginUser = asyncHandler(async (req, res, next) => {
           name: user.name,
           email: user.email,
         },
+        profile,
+
```

Comment:
Look the user is already loaded and sent to using the endpoint `/auth/user` and at the ui it's fetched using the function `loginWithCookies`.  the fix was in my opinion is make sure its visible by the user through `/profile/settings`. Would mind thinking about this from the ui's side not the back